src/tests/ftest/util/mdtest_utils.py

```diff
@@ -87,7 +87,7 @@ def __init__(self):
         self.dfs_group = FormattedParameter("--dfs.group {}")
         self.dfs_destroy = FormattedParameter("--dfs.destroy", True)
         self.dfs_chunk = FormattedParameter("--dfs.chunk_size {}", 1048576)
-        self.dfs_oclass = FormattedParameter("--dfs.oclass {}", "SX")
+        self.dfs_oclass = FormattedParameter("--dfs.oclass {}", "S1")
```

Comment:
This might be okay, but we would need to make sure all other tests using mdtest either set `dfs_oclass` or still work as expected with the new default. I think tests _**probably do**_ set it, so we could just verify with:
```
Features: mdtest
```

Reply:
Thank you! I will test CI with "Features: mdtest" later.
I changed the default to "S1" due to "S1" is used by DFS in mdtest_hard.yaml, so DFS and POSIX-IL show same performance.

Reply:
Right, but if you set this in the config, it will override the default for the test:
```
mdtest:
  dfs_oclass: SOME_OCLASS
```
Anyway, a default of `S1` makes more sense and probably won't break anything.